### ISSUE TITLE
High accuracy temp sensor mcp9808

### ISSUE THREAD

**Issue by JiriHolubcik** [esphome/feature-requests#214]
**Describe the problem you have/What new integration you would like**

Is it possible to add a support for the following sensor:
https://www.google.com/url?q=http://ww1.microchip.com/downloads/en/DeviceDoc/25095A.pdf&sa=U&ved=2ahUKEwjk9qnj_4TiAhUEEVAKHWYaCCwQFjAAegQIARAB&usg=AOvVaw1VsrZxHHwwvn8KS6gqYNfF
This sensor uses I2C and is not supported by default
**Please describe your use case for this integration and alternatives you've tried:**

**Additional context**

Here is some library: https://platformio.org/lib/show/820/Adafruit%20MCP9808%20Library

**Comment by chard62** [comment]
Higher accuracy temperature sensor, works with I2C. All the other common sensors have been supported.

**Comment by tarbax** [comment]
does this work ?

**Comment by cbvicious** [comment]
What exactly are we waiting with for a merge? I have a thermostat set up that I'd love to move over to ESPHome, but it has an MCP9808

**Comment by chard62** [comment]
It is a nice module, but I did not have the energy to proceed. I was able to get some very nice pre-built bluetooth battery operated units from Xiaomi that do the trick for me. Already ESPHome integrated too. About $7 USD each.Richard
-------- Original message --------From: cbvicious <notifications@github.com> Date: 2020-10-20  8:03 PM  (GMT-08:00) To: esphome/feature-requests <feature-requests@noreply.github.com> Cc: chard62 <hydra62@shaw.ca>, Comment <comment@noreply.github.com> Subject: Re: [esphome/feature-requests] High accuracy temp sensor mcp9808 (#214) 
What exactly are we waiting with for a merge? I have a thermostat set up that I'd love to move over to ESPHome, but it has an MCP9808
